lessons/02-html-blocks/index.html

```diff
@@ -0,0 +1,35 @@
+<html>
+    <head>
+        <title>
+            ex 02
+        </title>
+    </head>
+    <body>
+        <h1>
+            SAMPLE
+            <h2>
+                <p>Example math 10<sup>4</sup></p>
+                <p>Example Footnote<sub>3</sub></p>
+                <p><strong>Notice me</strong></p>
+                <p><em>Implying something</em></p>
+                <p><strong><em>Can be combined</em></strong></p>
+            </h2>
+            
+        </h1>
```

Comment:
We don't want to nest any other block elements inside of any of the h1-h6's, we need to just keep it to text or links. Likewise for the h2 inside here, a `<p>` tag is not valid inside

This would be correct:

```html
<h1>SAMPLE</h1>
<h2>Another Heading</h2>
<p>Example math 10<sup>4</sup></p>
<p>Example Footnote<sub>3</sub></p>
<p><strong>Notice me</strong></p>
<p><em>Implying something</em></p>
<p><strong><em>Can be combined</em></strong></p>
```